### PR TITLE
Loading screen

### PR DESCRIPTION
**Pull request by b3an5**
added a gif on the loading screen and downloaded the images that are not in our dataset

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -15,7 +15,7 @@
 .app-title-area {
   height: 250px;
   opacity: 0.7;
-  background: url('https://cdn-s3.si.com/styles/marquee_large_2x/s3/images/philly-eagles-broncos-fans.jpg?itok=1pEiQnWa');
+  background: url('./images/philly-eagles-broncos-fans.jpg');
```
Comment (ericfitzsimons451) [inline src/App.css]: download for faster viewing?

File: src/App.js [hunk 2]
```diff
@@ -24,10 +25,12 @@ class App extends Component {
...
-      this.setState({
-        nflTeams: data.nflTeams,
-        isLoading: false
-      })
+      setTimeout(() => {
+        this.setState({
+          nflTeams: data.nflTeams,
+          isLoading: false
+        })
+      }, 1000);
```
Comment (ericfitzsimons451) [inline src/App.js]: Very cool delay.  Might be harder to test

Comment (geet084) [reply src/App.js]: It's going to be hard to test regardless, but the delay could potentially help

File: src/App.js [hunk 3]
```diff
@@ -40,7 +43,7 @@ class App extends Component {
   render() {
     if (this.state.isLoading) {
       return(
-        <div>...Loading, yo.  Wait a bit...</div>
+        <div className="loading-screen"> <img className="loading-image" src={loadingGif} /> </div>
```
Comment (geet084) [inline src/App.js]: Better!

**Pull request merged**